site/docs/reference/working-logs-stats.md

```diff
@@ -0,0 +1,62 @@
+# Working with logs and statistics
+
+Your [`logs` and `stats` collections](../concepts/advanced/logs-stats.md)
+are useful for debugging and monitoring catalog tasks.
+
+:::caution
+Access to statistics is still a work in progress. For now, this documentation deals strictly with logs.
+:::
+
+## Accessing logs
+
+You can access logs by materializing them to an external endpoint, or from the command line.
+
+For illustrative purposes, the following sections use the word counts example from the [Flow tutorial](../getting-started/flow-tutorials/hello-flow.md).
+
+### Accessing logs from the command line
+
+:::info Beta
+Enhancements to this workflow are coming soon.
+:::
+
+From the command line, you can access logs by printing the journals that comprise them.
+This workflow is ideal if you require logs to debug specific catalog tasks,
+as logs are divided into journals per partition, and each partition maps to a task.
+
+Say you work for Acme Co, and you want to print logs for the materialization `materialize-word-counts`.
+You need to find the name of the correct journal.
+Begin by printing a list of active journals in your environment:
+
+```console
+flowctl journals list
+```
+
+Each task in the catalog will have a journal in the `logs` collection beginning with `ops/acmeCo/logs`.
+Look for the name of the desired task.
+In this case, the journal you’re looking for is: `ops/acmeCo/logs/kind=materialization/name=acmeCo%2Fpostgres%2Fmaterialize-word-counts/pivot=00 `
+
+Print the contents of the journal to view the logs:
+
+```console
+flowctl journals read -l name=ops/acmeCo/logs/kind=materialization/name=acmeCo%2Fpostgres%2Fmaterialize-word-counts/pivot=00
+```
+
+### Accessing logs by materialization
+
+You can materialize your `logs` collection to an external system.
+This is typically the preferred message if you’d like to work with logs for all tasks; in other words, the entire collection.
+
+```yaml
+acmeCo/postgres/logs:
+  endpoint:
+    connector:
+      image: ghcr.io/estuary/materialize-webhook:dev
+      config:
+        address: my.imaginarywebhook.com
+  bindings:
+    - resource:
+        relativePath: /log/wordcount
+      source: ops/acmeCo/logs
+```
+
+If necessary, you can also add [partition selectors](../../concepts/materialization/#partition-selectors) to only materialize logs of specific tasks.
```

Comment:
I think it's worth adding a note to this section about excluding the logs for the materialization task itself. Otherwise, if the materialization connector logs something it could end up in an infinite loop materializing its own logs and logging about it. I think the example should show this exclusion, and just explain the reason for it in a comment or note. What this look like is:

```yaml
bindings:
  - resource:
      relativePath: /logEvents
    source: ops/acmeCo/logs
    # Exclude the logs of this materialization task in order to avoid an infinite loop.
    partitions:
      exclude:
        name: ['acmeCo/postgres/logs']
```